src/EditorFeatures/CSharpTest/ReverseForStatement/ReverseForStatementTests.cs

```diff
@@ -0,0 +1,647 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeRefactorings;
+using Microsoft.CodeAnalysis.CSharp.ReverseForStatement;
+using Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.CodeRefactorings;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.ReverseForStatement
+{
+    public class ReverseForStatementTests : AbstractCSharpCodeActionTest
```

Comment:
>ReverseForStatementTests [](start = 17, length = 24)

Do you think it is reasonable to add a test verifying that the for-loop body is preserved?
A test for the for-loop operator with a body without braces.

Reply:
I'm ok with the tests as is.  we really don't touch the body at all. Maybe i can add in a future PR?